support-frontend/assets/helpers/urls/externalLinks.ts

```diff
@@ -73,7 +71,6 @@ function buildParamString(
 }
 
 function getLegacyPaperAndDigitalLink(
```

Comment:
I'm wondering whether this `getLegacyPaperAndDigitalLink` function is even used anymore? It used to generate a link to point the depreacted subscribe site.

Reply:
Yeah I'll go an clean up the unused functions in this file

Reply:
done!